test/Invariants.t.sol

```diff
@@ -35,6 +38,7 @@ contract ConnectorInvariants is Test {
         return targetContracts_;
     }
 
+    //    todo(nuno): fix this
```

Comment:
What's this todo?

Reply:
My bad! I left it after I changed the mock setup to introduce the mock router and that broke the invariant tests so I left this comment and forgot to delete it.